src/django_management/settings.py

```diff
@@ -48,6 +54,7 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'allauth.account.middleware.AccountMiddleware',
```

Comment:
same here - comment for what this is for would be great ! sorry to be nitpick on that, just so we can understand why this was added.

Reply:
same herer - These are apps required for social auth. This is like a template that we get in Django's initial setup.

Reply:
Same here - I think it'd be nice to include a comment saying that these are for social auth just to help keep things organized and explain what the different `middlewares` are for.

Also if you don't mind keep the comments unresolved until the original author reviews them, I don't see your responses once you resolve

Reply:
Done